### PR TITLE
[DO NOT MERGE] Update Microsoft.VMwareCloudSimple provider schema to 48 release of Cloudsimple

### PR DESCRIPTION
**Pull request by mpoliako**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/examples/GetFolder.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "api-version": "2019-04-01",
+    "subscriptionId": "{subscription-id}",
```
Comment (fengzhou-msft) [inline specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/examples/GetFolder.json]: The value should match the one in id, which is `b85c4986-56ae-4ebd-b5c5-a1595ca3dab1`

**Comment by fengzhou-msft** [comment]
What's the status of this PR? Are we waiting on something?

**Comment by mpoliako** [comment]
> What's the status of this PR? Are we waiting on something?

Sorry for late response, was in vacation, 
We are waiting for Cloudsimple release to production, this should happen in next 1-2 weeks.
Then PR can be merged

Thank you

**Comment by fengzhou-msft** [comment]
Any update for this PR?

**Comment by fengzhou-msft** [comment]
Close due to inactivity. Feel free to reopen when ready for next move.

**Pull request closed without merging**